app.js

```diff
@@ -3,14 +3,32 @@ const app = express();
 
 const { getTopics, getEndpoints } = require("./controllers/topics.controllers");
 
+const { getArticleId } = require("./controllers/articles.controllers");
+
 app.get("/api/topics", getTopics);
 
 app.get("/api", getEndpoints);
 
-app.use((err, req, res, next)=>{
-    if(err.msg && err.status){
-        res.status(err.status).send({msg: err.msg})
-    }
-})
+app.get("/api/articles/:article_id", getArticleId);
+
+app.use((err, req, res) => {
+  if (err.msg && err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  }
```

Comment:
You may want an else next(err) here to pass the error to the next middleware, might not be a problem now but could be in the future.